sdk/storage/Azure.Storage.Blobs.Cryptography/src/ClientsideEncryptionOptions.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Azure.Core.Cryptography;
+
+namespace Azure.Storage.Blobs.Specialized
+{
+    /// <summary>
+    /// Provides the client configuration options for connecting to Azure Blob using clientside encryption.
+    /// </summary>
+    public class ClientsideEncryptionOptions
```

Comment:
`Clientside` reads a little weird to me (should that be two words?).  What did they call this in Track 1/Java Track 2?  Maybe something morel like `EncryptedBlobOptions` to match the `EncryptedBlobClient` name?

Reply:
It could be two words. Using the track 1 name is not necessarily the best metric, as CPK didn't exist in the track 1 world. A few names have been selected to be distinct from CPK naming, as some model names for that feature use a generic "Encryption_____" naming pattern. I'll start a thread offline for this naming decision.